java/src/org/openqa/selenium/remote/TracedCommandExecutor.java

```diff
@@ -44,7 +44,7 @@ public Response execute(Command command) throws IOException {
       Map<String, ?> parameters = command.getParameters();
       if (parameters != null && parameters.size() > 0) {
         for (Map.Entry<String, ?> parameter : parameters.entrySet()) {
-          commandSpan.setAttribute("parameter." + parameter.getKey(), parameter.getValue().toString());
+          commandSpan.setAttribute("parameter." + parameter.getKey(), String.valueOf(parameter.getValue()));
```

Comment:
if `parameter.getValue()` is null, that will still throw a Null Pointer, unless mistaken.

if you're looking at ending up with `"null"` you'd be better of with :

`(String) parameter.getValue()`

Reply:
>if parameter.getValue() is null, that will still throw a Null Pointer, unless mistaken.

but test was green :)

ok let's make it clear with `Objects.toString(parameter.getValue(), "null")`